prisma/migrations/20260410140000_downloadmetadata_composite_pk/migration.sql

```
@@ -0,0 +1,7 @@
+UPDATE "DownloadMetadata" SET "guildId" = '' WHERE "guildId" IS NULL;
```

SQLFluff (4.1.0)

[error] The 'WHERE' keyword should always start a new line.

(LT14)